pallets/loans/src/lib.rs

```diff
@@ -232,8 +231,9 @@ pub mod pallet {
 
     /// The timestamp of the last calculation of accrued interest
     #[pallet::storage]
-    #[pallet::getter(fn last_accrued_timestamp)]
-    pub type LastAccruedTimestamp<T: Config> = StorageValue<_, Timestamp, ValueQuery>;
+    #[pallet::getter(fn last_accrued_interest_time)]
+    pub type LastAccruedInterestTime<T: Config> =
```

Comment:
will we have migration? but this looks much more clear

Reply:
we don't need to migrate this for we just use new storage here.